proto/mls/api/v1/mls.proto

```diff
@@ -78,15 +78,47 @@ service MlsApi {
       body: "*"
     };
   }
+
+  // Query stored group messages
+  rpc QueryGroupMessages(QueryGroupMessagesRequest) returns (QueryGroupMessagesResponse) {
+    option (google.api.http) = {
+      post: "/mls/v1/query-group-messages"
+      body: "*"
+    };
+  }
+
+  // Query stored group messages
+  rpc QueryWelcomeMessages(QueryWelcomeMessagesRequest) returns (QueryWelcomeMessagesResponse) {
+    option (google.api.http) = {
+      post: "/mls/v1/query-welcome-messages"
+      body: "*"
+    };
+  }
+
+  // Subscribe stream of new group messages
+  rpc SubscribeGroupMessages(SubscribeGroupMessagesRequest) returns (stream xmtp.mls.message_contents.GroupMessage) {
+    option (google.api.http) = {
+      post: "/mls/v1/follow-group-messages"
```

Comment:
Small nit, any reason the endpoints are named differently from the protos? E.g. 'follow-group-messages' vs 'SubscribeGroupMessages' here and below, and 'publish' vs 'send' above

Reply:
Good catch!